src/components/frame/v5/pages/VerifiedPage/partials/Table/Table.tsx

```diff
@@ -13,7 +12,11 @@ import { useVerifiedTable } from './hooks';
 
 const displayName = 'v5.pages.VerifiedPage.partials.Table';
 
-const Table: FC<TableProps> = ({ list, onReputationSortClick }) => {
+const Table: FC<TableProps> = ({
+  list,
+  onReputationSortClick,
+  isSortedDesc,
+}) => {
   // @TODO: Add action for adding new member, removing user from whitelist, handle pagination and sorting
```

Comment:
what about this comment?

Reply:
There are no designs for adding/removing members